app/code/Magento/InventoryInStorePickupApi/Api/Data/PickupLocationInterface.php

```diff
@@ -18,26 +18,27 @@
  */
 interface PickupLocationInterface extends ExtensibleDataInterface
 {
-    const IS_PICKUP_LOCATION_ACTIVE = 'is_pickup_location_active';
-    const FRONTEND_NAME = 'frontend_name';
-    const FRONTEND_DESCRIPTION = 'frontend_description';
+    public const IS_PICKUP_LOCATION_ACTIVE = 'is_pickup_location_active';
+    public const FRONTEND_NAME = 'frontend_name';
+    public const FRONTEND_DESCRIPTION = 'frontend_description';
+    public const PICKUP_LOCATION_CODE = 'pickup_location_code';
 
     /**
-     * Get source code of Pickup Location
+     * Get source code of Pickup Location.
      *
      * @return string
      */
-    public function getSourceCode(): string;
+    public function getPickupLocationCode(): string;
 
     /**
-     * Get Pickup Location name
+     * Get Pickup Location Name.
```

Comment:
Does it mean that we should use dot (`.`) for all the StorePickup PhpDocs now?

Reply:
Yes.